test/features/cloudnative/init_containers.go

```diff
@@ -45,19 +49,21 @@ func checkInitContainers(sampleApp *sample.App) features.Func {
 
 			var oneAgentInstallContainer *corev1.Container
 
+			var initContainerName string
 			for _, initContainer := range podItem.Spec.InitContainers {
-				if initContainer.Name == webhook.InstallContainerName {
+				if initContainer.Name == oldInstallContainerName || initContainer.Name == webhook.InstallContainerName {
 					oneAgentInstallContainer = &initContainer // loop breaks after assignment, memory aliasing is not a problem
+					initContainerName = initContainer.Name
```

Comment:
`initContainerName` variable is not needed. It duplicates `oneAgentInstallContainer.Name`.

maybe `oneAgentInstallContainer` could be renamed to `oneAgentInstallInitContainer`

Reply:
Yeah make sense. Will fix

Reply:
@aorcholski  PTAL, fixed in 8a6e1fcc7